src/test/java/com/karankumar/bookproject/backend/entity/PredefinedShelfTest.java

```diff
@@ -43,7 +41,7 @@ void testPredefinedShelvesWithoutBooksShouldStillExist(
         List<PredefinedShelf> shelves = predefinedShelfService.findAll();
 
         // then
-        assertEquals(4, shelves.size());
+        assertThat(shelves).hasSize(4);
```

Comment:
Thanks for this. While you're here, could you use `SoftAssertions` here as well (`assertSoftly`)?